src/Service.Host/client/src/components/BomUtility/BomUtility.js

```diff
@@ -119,14 +119,19 @@ function BomUtility() {
     const onContextMenu = e => {
         e.preventDefault();
         const { target } = e;
-        const detail = selected.children.find(x => x.name === target.innerText);
+        const detail = selected.children.find(x => x.id === target.id);
         setContextMenu(
             crNumber && contextMenu === null
                 ? {
                       mouseX: e.clientX + 2,
                       mouseY: e.clientY - 6,
                       part: target.innerText,
-                      canReplace: Number(crNumber) !== detail.addChangeDocumentNumber,
+                      canDelete: !detail.deleteReplaceSeq,
+                      canReplace:
+                          Number(crNumber) !== detail.addChangeDocumentNumber &&
+                          !detail.addReplaceSeq &&
```

Comment:
does the !addReplaceSeq prevent chaining of change requests i.e. replace a component on one change request then change that change to another part number in another one.  not common but sometimes Jacki might do crazy stuff like that (worth asking)

Reply:
Yeah - as it stands you can delete or replace a change that is involved in a deletion/replacement on a different change request